app/assets/javascripts/wizard/components/font-preview.js

```diff
@@ -38,6 +38,10 @@ export default createPreviewComponent(305, 165, {
   paint(ctx, colors, font, width, height) {
     const headerHeight = height * 0.3;
 
+    if (font === "-apple-system") {
```

Comment:
I think I messed something up during initial implementation because at this point, `font` should hold the value of `font[:name]`.